test/test_namedtensor.py

```diff
@@ -272,6 +272,19 @@ def test_diagonal(self):
         self.assertEqual(named_tensor.diagonal(outdim='E', dim1='B', dim2='D').names,
                          ['A', 'C', 'E'])
 
+    def test_max_polling(self):
```

Comment:
We should test CUDA as well because the code path is different for CPU and CUDA.